scripts/test_object_models/run_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Short script to run all the tests
+
+for i in *_test.sh ; do
+    echo ${i}
+    ./$i
+    echo
```

Comment:
If we're actually going to run multiple tests, we should make sure that this script exits with non-zero status if _any_ of the tests fail.

Also, perhaps we should look at some language-agnostic testing framework. In wit we've also implemented a custom sh-based, testing framework: https://github.com/sifive/wit/tree/0.12.x/t

cc @jackkoenig @mmjconolly